CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This changelog only contains the changes that are unreleased. For changes for in
 
 ### New Features
 
+1. Add the option to join a minecraft server when launching a instance [#893](https://github.com/ATLauncher/ATLauncher/pull/893)
```

Comment:
> Linking to the issue and like `[#748]` instead of a full link here is good, we convert it to a full link during the release process
> ```suggestion
> - Add the option to join a minecraft server when launching a instance [#748]
> ```

I will update it